alphabase/psm_reader/sage_reader.py

```diff
@@ -292,7 +292,7 @@ def _lookup_modification(
         The name of the matched modification in alphabase format.
 
     """
-    mass_distance = mod_annotated_df["mass"].values - mass_observed
+    mass_distance = mod_annotated_df["mass"].to_numpy() - mass_observed
```

Comment:
This is a bit of a tricky change. It might have some performance implications as it performas a copy.

Reply:
If you wish, we can disable the rule and revert? however, pandas docs recommend using `to_numpy()` 
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.values.html